frontend/simple/views/containers/GroupPledgesGraph.vue

```diff
@@ -0,0 +1,236 @@
+        {{fakeStore.currency}}{{graphData.totalAmount}}
+      </graph-legend-item>
+
+      <graph-legend-item :label="L('Needed Pledges')" :class="legendItemClass" color="light"
+        v-if="graphData.neededPledges"
+      >
+        {{fakeStore.currency}}{{graphData.neededPledges}}
+      </graph-legend-item>
+
+      <graph-legend-item :label="L('Surplus (not needed)')" :class="legendItemClass" color="secondary"
+        v-if="graphData.surplus"
+      >
+        {{fakeStore.currency}}{{graphData.surplus}}
+      </graph-legend-item>
+
+      <graph-legend-item :label="L('Income to receive')" :class="legendItemClass" color="tertiary"
+        v-if="graphData.userIncomeToReceive"
+      >
+        <tooltip direction="right" v-if="graphData.userIncomeNeeded !== graphData.userIncomeToReceive">
+          {{fakeStore.currency}}{{graphData.userIncomeToReceive}}
+          <i class="fa fa-info-circle is-size-6 has-text-tertiary c-legendItem-icon"></i>
+
+          <template slot="tooltip">
+            <strong class="has-text-weight-bold">Income Incomplete</strong>
+            <i18n tag="p" class="has-text-weight-normal" :args="{ amount: `${fakeStore.currency}${graphData.userIncomeNeeded}` }">
+              The group at the moment is not pledging enough to cover everyone's mincome.
+              So you'll receive only a part instead of the {amount} you need.
+            </i18n>
+          </template>
+        </tooltip>
+        <template v-else>
+          {{fakeStore.currency}}{{graphData.userIncomeToReceive}}
+        </template>
+      </graph-legend-item>
+    </graph-legend-group>
+  </div>
+</template>
+<style lang="scss" scoped>
+@import "../../assets/sass/theme/index";
+
+.c-graph {
+  position: relative;
+  flex-wrap: wrap;
+  align-items: center;
+  align-content: flex-start;
+}
+
+.c-legend {
+  flex-basis: 50%;
+  flex-grow: 1;
+  padding: $gi-spacer 0 $gi-spacer $gi-spacer-lg;
+
+  &-item {
+    padding: $gi-spacer-sm;
+    flex-basis: 33.3%;
+
+    @include phone {
+      padding: $gi-spacer-sm $gi-spacer-xs;
+      flex-basis: 50%;
+    }
+
+    @include tablet {
+      flex-basis: 50%;
+    }
+  }
+
+  &-icon {
+    display: inline-block;
+    margin-left: $gi-spacer-xs;
+  }
+}
+</style>
+<script>
+import { PieChart, GraphLegendGroup, GraphLegendItem } from '../components/Graphs/index.js'
+import Tooltip from '../components/Tooltip.vue'
+import currencies from '../utils/currencies.js'
+import { debounce } from '../../utils/giLodash.js'
+
+export default {
+  name: 'GroupPledgesGraph',
+  components: {
+    PieChart,
+    GraphLegendGroup,
+    GraphLegendItem,
+    Tooltip
+  },
+  props: {
+    // Needed to have a dynamic graph (ex: used on Contributions page)
+    userPledgeAmount: {
+      type: Number,
+      default: null
+    },
+    userIncomeAmount: {
+      type: Number,
+      default: null
+    }
+  },
+  data () {
+    return {
+      // REVIEW - make this a watcher
+      isMobile: this.verifyIsMobile(),
```

Comment:
I'm not sure I understand what this `REVIEW` comment is saying? What would the watcher be for?

Reply:
TODO - use a global debounce that calls sbp to emit an event with width info.

Reply:
issue opened https://github.com/okTurtles/group-income-simple/issues/492 to address this issue in another PR